docs/src/main/sphinx/installation/cli.rst

```diff
@@ -134,11 +303,57 @@ JWT authentication
 To access a Trino cluster configured to use :doc:`/security/jwt`, use the
 ``--access-token=<token>`` option to pass a JWT to the server.
 
+.. _cli-kerberos-auth:
+
 Kerberos authentication
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-If your cluster is secured with Kerberos authentication, see
-:doc:`/security/cli`.
+In addition to the options that are required when connecting to an unauthorized
+Trino coordinator, invoking the CLI with Kerberos support enabled requires a
+number of additional command line options. The simplest way to invoke the CLI is
+with a wrapper script.
+
+.. code-block:: text
+
+    #!/bin/bash
+
+    ./trino \
+      --server https://trino-coordinator.example.com:7778 \
+      --krb5-config-path /etc/krb5.conf \
+      --krb5-principal someuser@EXAMPLE.COM \
+      --krb5-keytab-path /home/someuser/someuser.keytab \
+      --krb5-remote-service-name trino \
+      --keystore-path /tmp/trino.jks \
+      --keystore-password password \
+      --catalog <catalog> \
+      --schema <schema>
+
+The following table list the available options for Kerberos authentication:
+
+.. list-table:: CLI options for Kerberos authentication
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--krb5-config-path``
+    - Kerberos configuration files.
+  * - ``--krb5-credential-cache-path``
+    - Kerberos credential cache path.
+  * - ``--krb5-disable-remote-service-hostname-canonicalization``
+    - Disable service hostname canonicalization using the DNS reverse lookup.
+  * - ``--krb5-keytab-path``
+    - The location of the keytab that can be used to authenticate the principal
+      specified by ``--krb5-principal``.
+  * - ``--krb5-principal``
```

Comment:
While this list is ordered alphabetically maybe it'd be useful to order them by commonly used configs to more advanced ones? I don't have strong opinion since the list is short anyway.

Reply:
I think we should leave as is